CONTRIBUTING.md

```diff
@@ -72,7 +72,7 @@ The detailed descriptions of file entries provided below.
 
 **`description`**
 
-Description of the model. Must match with the description from the model [documentation](#documentation).
+Description of the model. Must match with the description from the model [documentation](#documentation). Use [this](./tools/description_updater/documentation_updater.py) script for easy update.
```

Comment:
I think you should put it in `/ci/` instead. It's not a user-facing tool. (And it'll be reasonable to plug it into CI in the future.)

Reply:
I'm disagree with that. I suppose this would be a developer helper tool.  Or you talking just about relocation?

Reply:
>  I suppose this would be a developer helper tool.

Exactly, so not for users. :-) We don't have a directory for developer tools; I suppose we could make one, but given that this could be of use in CI, I think it would be easier to just stick it into `/ci/`.